gallery/src/CodeExample.tsx

```diff
@@ -28,16 +28,14 @@ const removeLicense = (content: string) =>
 
 const CodeExample: FunctionComponent<Props> =
   function CodeExample({ content, language }): ReactElement<Props> {
-    const licenseStrippedContent = removeLicense(content),
-        headerLanguagePart = language ? ` (${language.toUpperCase()})` : '';
-
+    const licenseStrippedContent = removeLicense(content);
     return (
       <section className="gallery-example-code">
-        <h3 className="nx-h3 nx-tile__section-header">
-          Code Snippet
-          {headerLanguagePart}
-          :
-        </h3>
+        { language &&
+          <h3 className="nx-h3 nx-tile__section-header">
+            Code Snippet ({language.toUpperCase()}):
```

Comment:
Would it make more sense now if it just said the language name instead of preceding it with "Code Snippet"?

Reply:
> > Would it make more sense now if it just said the language name instead of preceding it with "Code Snippet"?
> 
> I'm fine with either. Any preference @entroform and @glenhunter ?

Removed here: 3447c8ab275c0221c3e3f4f17bea6f713895ef4a